api/board.go

```diff
@@ -30,11 +32,27 @@ func (client *Client) GetAllBoards(projectKeyOrID string, word string) ([]jira.B
 	}
 }
 
-func (client *Client) GetAllSprints(boardID string) ([]jira.Sprint, error) {
-	sprints, _, err := client.jira.Board.GetAllSprints(boardID)
+func (client *Client) GetBoardByName(name string) (*jira.Board, error) {
+	boards, err := client.GetAllBoards("", name)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, board := range boards {
+		if strings.TrimSpace(board.Name) == strings.TrimSpace(name) {
```

Comment:
https://developer.atlassian.com/cloud/jira/software/rest/#api-rest-agile-1-0-board-get

> name
> string
> Filters results to boards that match or partially match the specified name.

This tool need to get the board which has exact name.